Dockerfile.dapper

```diff
@@ -1,10 +1,19 @@
-FROM registry.suse.com/bci/golang:1.19-20.4
+FROM ghcr.io/oracle/oraclelinux:7-slim
```

Comment:
Shouldn't this be oraclelinux:8-slim instead?

Reply:
Not sure when we need to update the base image to 8-slim. I just used the OL7 similar to others.

Reply:
I believe it was decided to use oraclelinux:8-slim going forward.

Reply:
I checked with @mgianatagh and it appears we would need an epic to move all of our BFS builds to move to OL8. So right now Ol7 will stay for a while.